daemon/mgr/container.go

```diff
@@ -190,7 +188,7 @@ func (mgr *ContainerManager) CreateExec(ctx context.Context, name string, config
 func (mgr *ContainerManager) StartExec(ctx context.Context, execid string, config *types.ExecStartConfig, attach *AttachConfig) error {
 	v, ok := mgr.ExecProcesses.Get(execid).Result()
 	if !ok {
-		return fmt.Errorf("exec process: %s not found", execid)
+		return errors.Wrap(ErrNotfound, "exec process: "+execid)
```

Comment:
The error message shows like `exec process: 12345678: not found`.
How about eliminate one `:` to be `exec process 12345678: not found`?